sirepo/pkcli/static_files.py

```diff
@@ -28,29 +26,19 @@ def gen(target_dir):
 class _Gen(PKDict):
     def __init__(self, target_dir):
         self.tgt = pykern.pkio.py_path(target_dir)
-        self.count = PKDict(root=0)
         for r, s in sirepo.resource.static_files():
             self._copy(r, s)
-            self._maybe_root(r, s)
-
-        self._verify()
```

Comment:
Why did you remove this stuff?

Reply:
Move robots.txt generated file to static because nginx looks for it there.
Removed caching of robots.txt in server because it is referred to using the output from pkcli.static_files on nginx.
Removed copy and verification of favicons to root because nginx refers to the files in static/img already.